openedx/frontend-app-publisher
# feat: add last modified time to edit course form

[PROD-3301](https://2u-internal.atlassian.net/browse/PROD-3301)

## Description:
This PR add the last modified time to edit course form

### Screenshot
<img width="963" alt="image" src="https://user-images.githubusercontent.com/78806673/232341986-80094681-88d7-4513-9fa1-ab99ac4e5182.png">

### Testing Instructions:
 - Pull backend PR changes from here https://github.com/openedx/course-discovery/pull/3867
 - Run `discovery`, `studio` and  `frontend-app-publisher`
 - Edit any of the field of course and verify that the last modified timestamp updated without refreshing
